DotnetCLIVersion.txt

```diff
@@ -1 +1 @@
-2.1.301
+2.1.401
```

Comment:
This was required to resolve symbol publish issue. https://github.com/dotnet/core-setup/issues/4291